.yamato/scripts/update_from_upstream.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# Prequisites:
+# This script assumes git and github CLR (gh) are installed.
+
+# Fetch from upstream and rebase our local main on it
+git remote add upstream https://github.com/dotnet/runtime
```

Comment:
Do we need error checking for all these commands?

Reply:
Good point - I think we need to add `set -e` at the top of this file to get that behavior.